etc/config/swift.amazon.properties

```diff
@@ -24,16 +36,22 @@ compiler.swiftdevsnapshot.semver=devsnapshot
 # Release Compilers
 #################################
 
-# 6.x
+# 6.x (x86)
 
 compiler.swift603.exe=/opt/compiler-explorer/swift-6.0.3/usr/bin/swiftc
 compiler.swift603.semver=6.0.3
 
+# 6.x (arm)
+
+compiler.arm-swift603.exe=/opt/compiler-explorer/swift-6.0.3/usr/bin/swiftc
+compiler.arm-swift603.semver=6.0.3
+compiler.arm-swift603.options=-target aarch64-swift-linux-musl -sdk /opt/compiler-explorer/swift-6.0.3-static-sdk/swift-linux-musl/musl-1.2.5.sdk/aarch64 -resource-dir /opt/compiler-explorer/swift-6.0.3-static-sdk/swift-linux-musl/musl-1.2.5.sdk/aarch64/usr/lib/swift_static
+
 # 5.x
 
 compiler.swift510.exe=/opt/compiler-explorer/swift-5.10/usr/bin/swiftc
 compiler.swift510.semver=5.10
-compiler.swift510.alias=switf510nightly
+compiler.swift510.alias=swift510nightly
```

Comment:
i originally opened https://github.com/compiler-explorer/compiler-explorer/pull/7435 to address this independently since it's not logically related to the other changes here, but to avoid potential merge conflicts decided to combine them.

Reply:
Thanks! Good spot!!